File: patterns/c-patterns/pattern67.c
// Pattern 67: ...

// *****5
// ****4
// ***3
// **2
// *1
// 0

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;
    for (i = n; i >= 0; i--)
    {
        for (j = 0; j <= i; j++)
        {
            if (i > j)
                printf("*");
            else
                printf("%d", i);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern67.js
// Pattern 67: ...

// *****5
// ****4
// ***3
// **2
// *1
// 0

const n = 5;

for (let i = n; i >= 0; i--) {
    let row = '';

    for (let j = 0; j <= i; j++) {
        if (i > j) {
            row += '*';
        } else {
            row += i;
        }
    }

    console.log(row);
}
